### DISCUSSION TITLE
Improve wording: share (should be internal) vs. publish (should be public)

### DISCUSSION THREAD

**Discussion by EduNetEurope** [outline/outline#6381] [Ideas]
The wording should be improved. It is very confusing that _publish_ = 'share internally' and _share_ = 'publish to internet'.

**Comment by tommoor** [comment]
Agreed, we plan to take a look at how drafts work soon. The share UI is also being rebuilt right now to allow sharing of individual documents and that will clarify a little bit

**Reply by EduNetEurope** [reply]
At this point sharing a document with nested documents ist an all-or-nothing option.
Will the rebuilt share UI add the ability to "not" share publicly selected documents nested under a shared document?

**Reply by paulux84** [reply]
i've opened an "idea" about draft that i think can add a really usefull feature . Take a look at #6728
